contextutil/context.go

```diff
@@ -48,8 +48,8 @@ func Tenant(ctx context.Context) (config.TenantConfiguration, error) {
 	return tc, nil
 }
 
-// CentContext updates a context with tenant info using the configstore, must only be used for api handlers
-func CentContext(ctx context.Context, cs config.Service) (context.Context, error) {
+// NodeContext updates a context with tenant info using the configstore, must only be used for api handlers
+func NodeContext(ctx context.Context, cs config.Service) (context.Context, error) {
```

Comment:
I am not certain about this renaming, as Node sounds like a static context to me, and the purpose of this context is on request basis. I think it makes sense to just use `Context(...)`.